docker-compose/pre-release-docker-compose.yml

```diff
@@ -197,7 +197,7 @@ services:
       APP_CONFIG_PLUGIN_MICROSOFT_ONLINE: "false"
       APP_BASE_SHARE_URL: "http://localhost:8080/workspace/#/preview/s"
   control-center:
-    image: quay.io/alfresco/alfresco-control-center:9.0.0-10627955718
+    image: quay.io/alfresco/alfresco-control-center:9.1.0-11106981658
```

Comment:
On quay I next version I see is 10.0.0-11108013258 - we can probably update the matrix for pre release

Reply:
matrix is already ok, it's just that I run the workflow before that specific repo started to build new alphas with the bumped version (they are doing it manually after the release happens)